### PR TITLE
docs: update alert provider types

### PR DESCRIPTION
**Pull request by moshloop**
Signed-off-by: Moshe Immerman <moshe@flanksource.com>

### CODE REVIEW

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @moshloop 🏅

File: docs/spec/v1beta1/provider.md [hunk 1]
```diff
@@ -8,81 +8,85 @@ Spec:
...
-	// +optional
-	Proxy string `json:"proxy,omitempty"`
-
-	// Secret reference containing the provider webhook URL
-	// +optional
-	SecretRef *meta.LocalObjectReference `json:"secretRef,omitempty"`
-
-	// CertSecretRef can be given the name of a secret containing
-	// a PEM-encoded CA certificate (`caFile`)
-	// +optional
```
Comment (stefanprodan) [inline docs/spec/v1beta1/provider.md:40]: Please undo this change, Go formatting is not optional for us.

**Review by stefanprodan — CHANGES_REQUESTED** [review]

File: docs/spec/v1beta1/provider.md (hunk 1 above)
Comment (moshloop) [reply docs/spec/v1beta1/provider.md]: sorry - my vscode auto-formats optimistically

**Review by stefanprodan — APPROVED** [review]
